ocs_ci/ocs/node.py

```diff
@@ -769,10 +769,14 @@ def get_ocs_nodes(num_of_nodes=None):
             it returns all the ocs nodes.
 
     Returns:
-        list: List of ocs nodes
+        list: List of ocs nodes. If not found, it returns None.
 
     """
     ocs_node_names = machine.get_labeled_nodes(constants.OPERATOR_NODE_LABEL)
+    if not ocs_node_names:
+        log.warning("Didn't find any OCS nodes")
```

Comment:
Why do we need this, we are not going to run any testcase without OCS nodes, This list can never be empty. Please correct me if my understanding is wrong

Reply:
I saw that the list is empty with external mode(4.10) and managed service. See this failure here: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster-prod/3887/testReport/tests.manage.z_cluster.nodes.test_check_pods_status_after_node_failure/TestCheckPodsAfterNodeFailure/test_check_pods_status_after_node_failure/.